features/contract-state/GeneralInfo.tsx

```diff
@@ -1,37 +1,38 @@
 import styled from "styled-components";
 import { utils } from "ethers";
 import { Typography, Box, Tooltip } from "@material-ui/core";
+import { useState, useEffect } from "react";
```

Comment:
You can remove `useState` and `useEffect` now right?

Reply:
ye, you right! have removed this & pushed up for the merge 🚀